tests/fixtures/make_sha1_hashes.js

```diff
@@ -0,0 +1,14 @@
+const crypto = require('crypto');
```

Comment:
"use strict"

Reply:
We can add an eslint rule for this as well!

Reply:
Ref: #41